NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -696,17 +700,30 @@ class NeuroID private constructor(
         return SessionIDOriginResult(origin, originCode, sessionID)
     }
 
-
+    @Synchronized
     fun pauseCollection() {
         isSDKStarted = false
-        CoroutineScope(Dispatchers.IO).launch {
+        pauseCollectionJob = CoroutineScope(Dispatchers.IO).launch {
```

Comment:
How will consecutive calls to `pauseCollection()` behave?  References to the jobs will be lost, is that ok?

Reply:
Good point. It looks like we will indeed run into the same issue when folks do this. If a pauseCollection() job is in progres, we should ignore subsequent calls until the existing pause is complete. I'll look into adding this. We will also add passive protection which will include guidelines on usage of the session methods (takes about 700ms on average to complete, don't call pauseCollection() faster than that, etc) but this will not help folks who don't read/follow docs.

Reply:
updated to ignore new calls it pauseCollection() until the first one is done.